src/assets/TestAutomationWithGo-ABlogSeries/goBasics.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Go Basics - The Starting Point
+authorName: Prarup Gurung(प्रारुप गुरुङ्ग)
+authorAvatar: https://www.jankaritech.com/images/2021/12/29/p1110938-crop-colour-800.jpg
+authorLink: https://github.com/grgprarup
+createdAt: Dec 10, 2022
+tags: golang, gopher, go, test automation, testing, bdd, godog
+banner: https://camo.githubusercontent.com/2b507540e2681c1a25698f246b9dca69c30548ed66a7323075b0224cbb1bf058/68747470733a2f2f676f6c616e672e6f72672f646f632f676f706865722f6669766579656172732e6a7067
+seriesTitle: Journey With Go - A Blog Series
+episode: 1
+---
+
+**गो** **"Journey With Go - A Blog Series"** about the basics of Go `Go Basic`, advanced concepts in Go `Go Beyond Basics`, testing with go `Godog as Test Framework` and many more. This is the first blog in the series. In this blog, we will learn about installing Go, and initializing a Go project. We will also learn about the basic data types, variables, and constants in Go. So, let's get started.
+
+## Introduction
+- Designed and Developed at Google by Robert Griesemer, Rob Pike, and Ken Thompson
+- Go is an open-source programming language supported by Google
+- Go is statically typed i.e. you need to declare the type of a variable before using it
+- Go is compiled programming language i.e. you need to compile your code before running it
+
+## Why Go?
+
+- Free and open source
+- Automatic Garbage Collection
+- Richer built-in types
+- Robust standard library
+- Built-in support for concurrency
+- Support function multiple return values
+- Beautiful error handling
```

Comment:
```suggestion
- Better error handling
```
or other words can be given?

Reply:
IMO this `Beautiful` word gives more weightage to the statement.